dubbo-configcenter/dubbo-configcenter-etcd/src/main/java/org/apache/dubbo/configcenter/support/etcd/EtcdDynamicConfiguration.java

```diff
@@ -179,7 +174,6 @@ private void watch() {
                     .setKey(ByteString.copyFromUtf8(key))
                     .setProgressNotify(true);
             WatchRequest req = WatchRequest.newBuilder().setCreateRequest(builder).build();
-            observer.onNext(req);
```

Comment:
Why remove this line?

Reply:
Oops! this is an unexpected deletion. I will add it back.